src/main/java/uk/gov/homeoffice/digital/sas/accruals/model/Agreement.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.homeoffice.digital.sas.accruals.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.OneToMany;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+import uk.gov.homeoffice.digital.sas.accruals.enums.SalaryBasis;
+import uk.gov.homeoffice.digital.sas.accruals.enums.TermsAndConditions;
+import uk.gov.homeoffice.digital.sas.jparest.annotation.Resource;
+import uk.gov.homeoffice.digital.sas.jparest.models.BaseEntity;
+
+@Resource(path = "agreements")
+@Entity(name = "agreement")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Getter
+@Setter
+public class Agreement extends BaseEntity {
+
+  @NotNull(message = "Person ID should not be null")
+  @JdbcTypeCode(SqlTypes.CHAR)
+  private UUID personId;
+
+  @DecimalMin(value = "0.0", inclusive = false)
+  @Digits(integer = 1, fraction = 4)
+  @DecimalMax(value = "1.0")
+  private BigDecimal fteValue;
```

Comment:
Similar concern here as expressed in the database schema for agreement

Reply:
Addressed. See comment above